docs/Common/Structs/index.md

```diff
@@ -468,6 +468,9 @@
 |moduleType|Common.ModuleType|true||The moduleType indicates which type of data should be changed and identifies which data object exists in this struct. For example, if the moduleType is CLIMATE then a "climateControlData" should exist|
 |radioControlData|Common.RadioControlData|false|||
 |climateControlData|Common.ClimateControlData|false||
+|audioControlData|Common.AudioControlData|false|||
```

Comment:
Extra `|`, might be causing the parsing failure

Reply:
@jacobkeeler, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/103/commits/15bd7c4a57a236266a14c48d32cb63372fe12e36